src/tools/rbd/Utils.cc

```diff
@@ -806,10 +806,10 @@ int get_image_size(const boost::program_options::variables_map &vm,
 
 int get_path(const boost::program_options::variables_map &vm,
              const std::string &positional_path, std::string *path) {
```

Comment:
You should replace ```const std::string& positional_path``` with a ```size_t* arg_index``` parameter so that it directly grabs the positional argument and then increments the ```arg_index```.